src/UI/templates/default/Glyph/glyph.less

```diff
@@ -8,6 +8,12 @@
 		}
 	}
 
+	&.disabled {
+		opacity: 0.2;
+		filter: alpha(opacity=20); /* For IE8 and earlier */
```

Comment:
No need to support IE8.

I guess a color: @il-glyph-disabled-color; would make more sense than using opacity.